operations/boot_orchestration/BOS_export_import.md

```diff
@@ -0,0 +1,66 @@
+# BOS - Exporting and Importing for System Recovery or in the case of a fresh install
+
+## BOS Session Templates
+
+The primary BOS data that should be saved is the BOS session template. BOS session templates can be exported and later imported.
+
+### Automatically export/import BOS session templates
+
+Product specified session templates can be re-installed using their associated Helm charts. This recreates the session template data within BOS.
+Existing BOS session templates can be backed up and restored using the automated scripts. See scripts/operations/system_recovery/export_bos_sessiontemplates.sh and scripts/operations/system_recovery/import_bos_sessiontemplates.sh
+To export BOS session templates, simply run the script scripts/operations/system_recovery/export_bos_sessiontemplates.sh. Copy the archive file it outputs to a safe location.
+To import the BOS session templates, run the scripts/operations/system_recovery/import_bos_sessiontemplates.sh providing as input the archive file that was created by the export_bos_sessiontemplates.sh script.
+
+BOS session templates can also be manually exported and imported onto a given system using the Cray CLI tool. For each session template that you wish to export, use `cray bos sessiontemplate describe` cli. For example:
+
+### Manually export any session template as needed
+
+```bash
+ncn# cray bos sessiontemplate describe uan-sessiontemplate-2.0.27 --format json > ~/uan-sessiontemplate-2.0.27.json
```

Comment:
Please remove all the prompts to facilitate copy-paste.

Reply:
Is there a docs writing guidance that I should be reading?

Reply:
Despite my protests, I fixed this.